setup/gcp/installer/forseti_server_installer.py

```diff
@@ -188,7 +186,8 @@ def get_configuration_values(self):
             'EMAIL_SENDER': self.config.notification_sender_email,
             'SENDGRID_API_KEY': self.config.sendgrid_api_key,
             'SCANNER_BUCKET': bucket_name[len('gs://'):],
-            'DOMAIN_SUPER_ADMIN_EMAIL': self.config.gsuite_superadmin_email
+            'DOMAIN_SUPER_ADMIN_EMAIL': self.config.gsuite_superadmin_email,
```

Comment:
I think the key here should be 'GSUITE_ADMIN_EMAIL' instead, that's what we are using in our deployment template.

Reply:
Here is the line in the deployment template:

domain_super_admin_email: {DOMAIN_SUPER_ADMIN_EMAIL}

Reply:
Fixed by removing GSUITE_ADMIN_EMAIL from deployment template.